src/server/helpers/collections.js

```diff
@@ -104,6 +119,16 @@ export async function getOrderedPublicCollections(from, size, entityType, orm) {
 
 	const collectionsJSON = allCollections ? allCollections.toJSON() : [];
 
+	// We need to add the no. of entities in each collection
+	collectionsJSON.forEach((collection) => {
+		collection.noOfEntities = getCollectionSize(collection.id, orm);
+	});
+	// Resolving the promises
+	await Promise.all(collectionsJSON.map(async collection => {
+		const resolvedValue = await collection.noOfEntities;
+		collection.noOfEntities = resolvedValue;
+	}));
```

Comment:
After some thought, I think we can do this in a more optimized manner on the ORM side.
In the [UserCollection model](https://github.com/bookbrainz/bookbrainz-data-js/blob/master/src/models/userCollection.js), we should be able to add something like that:
```
itemCount() {
	return this.hasMany('UserCollectionItem', 'collection_id').count();
},
```

And then when fetching the collections, we can request our new `itemCount`:
```
const allCollections = await new UserCollection()
.where(…
.fetchPage({
	limit: size,
	offset: from,
	withRelated: ['owner', 'itemCount'] // <-- Adding it here
});
```

That way, the ORM will only have to do a COUNT  SQL query in the same transaction which should be faster, while also making it reusable in other parts of the codebase.

Now… we install our ORM bookbrainz-data-js as a package from NPM.
To test your changes locally there's a few more steps:
1. you'll need to clone the repo locally
2. from `bookbrainz-data-js`, run the command `npm link`
3. once that is done, from `bookbrainz-site` run `npm link bookbrainz-data`

What that does is create a local NPM package and substitutes it in the website repo.
With that, during development when you make changes to `bookbrainz-data-js` you can simply rebuild (`npm run build-js`) and then restart the web server.